manifests/s/SAGAUserGroupAssociation/SAGAGIS/8.1.0.0/SAGAUserGroupAssociation.SAGAGIS.installer.yaml

```diff
@@ -16,7 +16,7 @@ UpgradeBehavior: install
 Installers:
 - Architecture: x64
   InstallerUrl: https://sourceforge.net/projects/saga-gis/files/SAGA%20-%208/SAGA%20-%208.1.0/saga-8.1.0_x64_setup.exe/download
-  InstallerSha256: AD68CA00D56EB4049D95765DDA85C87B3294EDDBE7D2E5D51E2AEF29FD3F5200
+  InstallerSha256: C0D116BB8C1E6D7C78196E3335C803DA7A43248607545704F08B0C112D434A6F
```

Comment:
I'll let this one sit for a bit since it picked up the previous hash
```
12/21/2021 7:24:53 PM
Status: Completed
##[error] Installer https://sourceforge.net/projects/saga-gis/files/SAGA%20-%208/SAGA%20-%208.1.0/saga-8.1.0_x64_setup.exe/download
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   AD68CA00D56EB4049D95765DDA85C87B3294EDDBE7D2E5D51E2AEF29FD3F5200
Expected hash: c0d116bb8c1e6d7c78196e3335c803da7a43248607545704f08b0c112d434a6f
```